riot.im/release/conf_distributions

```diff
@@ -1,3 +1,10 @@
+Origin: riot.im
+Codename: default
```

Comment:
Hmm, I guess `Suite` would make more sense here...?

Reply:
Ah, but the develop builder checks only for `Codename` lines... Well, it's probably fine as-is then.

Reply:
Also I think you would always specify a Codename but may also specify a Suite? Not exactly sure though.

Reply:
Well, examples like `xenial` below seem to suggest you can get away with just `Codename` at least, so maybe this is fine...? Maybe. 😓